pom.xml

```diff
@@ -1,11 +1,19 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!-- ~ JBoss, Home of Professional Open Source. ~ Copyright 2010, Red Hat, Inc., and individual contributors ~ as indicated by the @author tags. 
-    See the copyright.txt file in the ~ distribution for a full listing of individual contributors. ~ ~ This is free software; you can redistribute 
-    it and/or modify it ~ under the terms of the GNU Lesser General Public License as ~ published by the Free Software Foundation; either version 
-    2.1 of ~ the License, or (at your option) any later version. ~ ~ This software is distributed in the hope that it will be useful, ~ but WITHOUT 
-    ANY WARRANTY; without even the implied warranty of ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU ~ Lesser General Public 
-    License for more details. ~ ~ You should have received a copy of the GNU Lesser General Public ~ License along with this software; if not, write 
-    to the Free ~ Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA ~ 02110-1301 USA, or see the FSF site: http://www.fsf.org. -->
+<!--
+Copyright [2023] [Red Hat, Inc.]
```

Comment:
These headers have a copyright part and a license part.

I'm double-checking whether changing the copyright clause is ok. I don't think so. It's not an aspect of changing to ASL.

Reply:
I've gotten feedback. Please just change this in all files you've modified to:

"Copyright Red Hat"